src/Services/FeatureChecker.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Supervisorg\Services;
+
+use Puzzle\Configuration;
+
+class FeatureChecker
+{
+    private
+        $configuration;
+
+    public function __construct(Configuration $configuration)
+    {
+        $this->configuration = $configuration;
+    }
+
+    public function isEnabled($name)
+    {
+        return $this->configuration->readRequired("features/$name/enabled");
```

Comment:
Hard to say that all features will be enabled or disabled by default.

So default behaviors are handled in the `config/built-in/feature.yml` file.